examples/seismic/skew_self_adjoint/README.md

```diff
@@ -44,4 +44,98 @@ These notebooks first implement and then test for correctness for three types of
 
 ## Running unit tests
 - if you would like to see stdout when running the tests, use
-```py.test -c testUtils.py```
\ No newline at end of file
+```py.test -c testUtils.py```
+
+## TODO
+- [ ] Devito-esque equation version of setup_wOverQ
+- [ ] figure out if the JacobianAdjointOperator completely solves p0 first
+- [ ] figure out if can get time sampling from the SparseTimeFuntions src/rec
+- [ ] replace the conditional logic in the stencil with comprehension
+- [ ] p --> u in all equations
+- [ ] \Gamma --> P_r,P_s
+- [ ] Add checkpointing back to the iso wavesolver
+- [ ] Farfield similarity tests for correctness, ensure 10 wavelengths out that wavelet phase is preserved
+- [ ] Add memoized methods back to wavesolver.py
+- [ ] Add ensureSanityOfFields methods for iso, vti, tti
+- [ ] Add timing info via logging for the wOverQ setup, as in initialize_damp
+- [ ] Add smoother back to setup_WOverQ method
+- [ ] Correctness tests
+  - [ ] Analytic response in the far field
+  - [X] Modeling operator linearity test, with respect to source
+  - [X] Modeling operator adjoint test, with respect to source
+  - [X] Nonlinear operator linearization test, with respect to model/data
+  - [X] Jacobian operator linearity test, with respect to model/data
+  - [X] Jacobian operator adjoint test, with respect to model/data
+  - [X] Skew symmetry test for shifted derivatives
+
+## To save generated code 
+
+```
+f = open("operator.c", "w")
+print(op.ccode, file=f)
+f.close()
+```
+
+## Some commands for performance testing thread scaling on AMD 7502
```

Comment:
Do we want these here. These are for a very specific hardware most people don't have access to.

Reply:
left over from my testing ... will remove